core/kubernetes/main/k8s-mini.sh

```diff
@@ -6,6 +6,9 @@ if [ "$1" == "up" ]; then
     minikube addons enable ingress
     kubectl apply -k $kustomizationFilePath
 
+    kubectl get services
+    kubectl get ingress
```

Comment:
Are these just for display purposes?

Reply:
Yeah - but they're actually not that useful at this point